code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -473,7 +473,7 @@
 	desc = "A curse that firmly attaches a demonic duffel bag to the target's back. The duffel bag will make the person it's attached to take periodical damage if it is not fed regularly, and regardless of whether or not it's been fed, it will slow the person wearing it down significantly."
 	spell_type = /obj/effect/proc_holder/spell/pointed/duffelbagcurse
 	category = "Assistance"
-	cost = 1
+	cost = 2
```

Comment:
oh, come on

Reply:
Yeah I'm thinking it's too much as well. Bringing it back down to 1.